akka-stream-tests/src/test/scala/akka/stream/scaladsl/SinkSpec.scala

```diff
@@ -206,6 +206,7 @@ class SinkSpec extends StreamSpec with DefaultTimeout with ScalaFutures {
   "The ignore sink" should {
 
     "fail its materialized value on abrupt materializer termination" in {
+      @silent("deprecated")
       val mat = ActorMaterializer()
```

Comment:
Don't we have a system in scope already so it can just be removed instead of silenced?